notification/telegram.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2016, Artem Feofanov <artem.feofanov@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+DOCUMENTATION = """
+
+module: telegram
+version_added: "2.1"
+author: "Artem Feofanov (@tyouxa)"
+
+short_description: module for sending notifications via telegram
+
+description:
+    - Send notifications via telegram bot, to verifyed group or user
+notes:
+    - You will require a telegram account and create telegram bot to use this module.
+options:
+  msg:
+    description:
+      - What message you wish to send.
+    required: true
+  token:
+    description:
+      - Token identifying your telegram bot.
+    required: true
+  chat_id:
+    description:
+      - Telegram group or user chat_id
+    required: true
+
+"""
+
+EXAMPLES = """
+
+send a message to chat in playbook
+- telegram:  token=bot9999999:XXXXXXXXXXXXXXXXXXXXXXX
+    chat_id=000000
+    msg="Ansible task finished"
+
+"""
+
+RETURN = """
+
+msg:
+  description: What message you wish to send
```

Comment:
The tense feels wrong here, would it be better as:

"The message you attempted to send"